### PR TITLE
Fixes reload

### LINKED ISSUE #12375: Reload is not working
### Description
Reload in packaged builds is not working
- Cmd + R
- Clicking reload button
- Right click reload button, choose reload context menu
- etc

(originally captured with https://github.com/brave/browser-laptop/issues/10354)

Root cause can be found here:
https://github.com/brave/browser-laptop/blob/874572770649aefc63dc421274b2eab472fc3544/app/common/actions/tabActions.js#L12-L13

When packaged build happens, the name is lost and it will try to execute "reload" instead of "tabActionsReload"

### Steps to Reproduce

  1. Launch Brave 0.20.9
  2. Load page
  3. Click reload button

**Actual result:**

Nothing happens

**Expected result:**
Page should reload

**Reproduces how often:**

100%

### Brave Version
0.20.9

**about:brave info:**

**Reproducible on current live release:**

### PR DESCRIPTION
**Pull request by NejcZdovc**
Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

Resolves #12375

Auditors:

Test Plan:
- build package
- run build brave
- check if reload is working

Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/common/actions/tabActions.js [hunk 1]
```diff
@@ -26,8 +27,8 @@ const tabActions = {
     })
   },
 
-  didStartNavigation: function tabActionsDidStartNavigation (tabId, navigationState, windowId) {
-    dispatchAction(tabActions.didFinishNavigation.name, {
+  didStartNavigation: (tabId, navigationState, windowId) => {
+    dispatchAction(tabActionConstants.START_NAVIAGTION, {
```
Comment (cezaraugusto) [inline app/common/actions/tabActions.js]: typo

File: app/common/constants/tabAction.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ module.exports = {
+   RELOAD: 'tabActionsReload',
+   FINISH_NAVIGATION: 'tabActionsDidFinishNavigation',
+   START_NAVIAGTION: 'tabActionsDidStartNavigation'
```
Comment (cezaraugusto) [inline app/common/constants/tabAction.js]: typo

File: app/browser/reducers/tabsReducer.js [hunk 3]
```diff
@@ -48,8 +48,8 @@ const getWebRTCPolicy = (state, tabId) => {
 const tabsReducer = (state, action, immutableAction) => {
   action = immutableAction || makeImmutable(action)
   switch (action.get('actionType')) {
-    case tabActions.didStartNavigation.name:
-    case tabActions.didFinishNavigation.name:
+    case tabActionConsts.FINISH_NAVIGATION:
+    case tabActionConsts.START_NAVIAGTION:
```
Comment (cezaraugusto) [inline app/browser/reducers/tabsReducer.js]: typo

File: test/unit/app/browser/reducers/tabsReducerTest.js [hunk 4]
```diff
@@ -173,7 +173,7 @@ describe('tabsReducer unit tests', function () {
       }
 
       const action = {
-        actionType: tabActions.didStartNavigation.name,
+        actionType: tabActionsConsts.START_NAVIAGTION,
```
Comment (cezaraugusto) [inline test/unit/app/browser/reducers/tabsReducerTest.js]: ditto

File: app/common/actions/tabActions.js (hunk 1 above)
Comment (NejcZdovc) [reply app/common/actions/tabActions.js]: fixed

File: app/common/constants/tabAction.js (hunk 2 above)
Comment (NejcZdovc) [reply app/common/constants/tabAction.js]: fixed

File: app/browser/reducers/tabsReducer.js (hunk 3 above)
Comment (NejcZdovc) [reply app/browser/reducers/tabsReducer.js]: fixed

File: test/unit/app/browser/reducers/tabsReducerTest.js (hunk 4 above)
Comment (NejcZdovc) [reply test/unit/app/browser/reducers/tabsReducerTest.js]: fixed

**Comment by bsclifton** [comment]
master 3406c21ef5a67fea84a486b70b40b37b920625b8
0.21.x 9c7756bd2826463dee6869b38aed43b43306533e
0.20.x 7af1900a2c31a487524d5e1edcb353a036b760b5

**Pull request merged**